docs/running.rst

```diff
@@ -515,6 +515,105 @@ They are summarized below:
 
 * ``--max-retries NUM``: Specify the maximum number of times a failed regression test may be retried (default: 0).
 
+
+
+Generating a Performance Report
+------------------------------
+
+A report of the performance values extracted during the regression tests can be generated by running ReFrame with the command line option `--performance-report`.
+
+The performance report is printed after the output of the regression and has the following format.
```

Comment:
The performance report is printed after the output of the regression tests and has the following format: